HISTORY.md

```diff
@@ -1,5 +1,11 @@
 # Version History
 
+## 0.4.0_preview / 2021-08-05
+
+- Added support for managing ACLs on securable collections
+- Support for additional collections (Users, Topics, Subscriptions, etc.)
+- Split related collections out of the Assts and Streams classes 
```

Comment:
@derekendres , you mentioned in the java library PR recently that breaking changes are good to note in the history file. do we want to label these as 'breaking' or is just calling it out like this sufficient. I'm fine with it the way it is, personally.